forge-ai/src/main/java/forge/ai/AiAttackController.java

```diff
@@ -784,9 +785,10 @@ private Pair<Integer, Integer> getDamageFromBlockingTramplers(final List<Card> b
             for (Card blocker : remainingBlockers) {
                 if (CombatUtil.canBlock(attacker, blocker) && damage > 0) {
                     damage -= ComputerUtilCombat.shieldDamage(attacker, blocker);
-                    remainingBlockers.remove(blocker);
+                    removedBlockers.add(blocker);
```

Comment:
@Hanmac does this also work? just found this before merging the fix.

Reply:
Yeah, that should fix it, i think